### PR TITLE
glibc: backport patch to fix CVE-2023-4911

### PR DESCRIPTION
**Pull request by Johnnynator**
https://lwn.net/ml/oss-security/20231003175031.GA16924@localhost.localdomain/

### CODE REVIEW

File: srcpkgs/glibc/patches/committed-1-2-Propagate-GLIBC_TUNABLES-in-setxid-binaries.patch [hunk 1]
```diff
@@ -0,0 +1,417 @@
+From patchwork Tue Oct  3 17:08:10 2023
```
Comment (classabbyamp) [inline srcpkgs/glibc/patches/committed-1-2-Propagate-GLIBC_TUNABLES-in-setxid-binaries.patch:1]: do we need to include all the email headers?

Comment (Johnnynator) [reply srcpkgs/glibc/patches/committed-1-2-Propagate-GLIBC_TUNABLES-in-setxid-binaries.patch]: Removed most of the headers.

File: srcpkgs/glibc/template [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'glibc'
 pkgname=glibc
 version=2.36
-revision=1
+revision=2
```
Comment (MarijnS95) [inline srcpkgs/glibc/template:4]: Just sanity-checking myself here: I'm doing a [custom rootfs build](https://github.com/SoMainline/void-bootstrap) courtesy of @JamiKettunen, and it's failing on `unresolved shlibs`. Should the revision suffix in `common/shlibs` have been bumped to `glibc-2.36_2`?

Comment (classabbyamp) [reply srcpkgs/glibc/template:4]: no

that error probably means there's something being installed that was removed from the repositories at some point

Comment (MarijnS95) [reply srcpkgs/glibc/template:4]: I did not want to spam a massive error message here for something that's likely broken on our end in either the script or our (dated!) package overlay(s).

In any case, substituting `glibc-2.36_1` for `glibc-2.36_2` in `common/shlibs` _fixes it_. I'll leave it to @JamiKettunen to sort out whether that's expected.

**Pull request merged**